slakh96/no-mans-land
# Created Preliminary Main Menu Screen

- Modified MainMenuScene 
- Created MainMenuScript.cs 
- Fixed all links for the Main Menu buttons 
TO TEST 
- Bring MainMenuScene to the hierarchy, ensure MainMenuCanvas is there 
- Bring DemoSceneFinal to the hierarchy 
- Click on File -> Build Settings -> Add Open Scenes 
- Ensure MainMenuScene has index 0 and DemoSceneFinal has index 1 
- Close the window and now try playing the game (make sure DemoSceneFinal is removed from hierarchy before playing. Only MainMenuScene should be in the hierarchy) 
- Main Menu's Quit button only works when you export the game, may not work in Unity window.
